src/test/resources/features/completion/wildcards_completion.feature

```diff
@@ -54,8 +60,11 @@ Feature: Completion for wildcards
           "{wildcard6}" '{wildcard7}' f"{{wildcard8}}"
           kwd="{wildcard6}"
       input: "{}"
+
```

Comment:
1) If I'm not mistaken wildcards set could be: set from overridden rule + set from URL, e.g.:
```
rule foo:
   log: "{sample}"
   
use rule foo as foo1:
  output: " {genome}" 
  input: "{<cursor>}"
```   
wildcards are inconsistent, but 2 of them are available in autocompletion in input:

2) What about overridden sections:
```
rule foo:
   log: "{sample}"
   message: "{wildcards.sample}"
   
use rule foo as foo1:
  output: " {donor}" 
  input: "{donor}"
```   
Seems `sample` shouldn't be available in `foo1` and more over we have undefined wildcards in `message`. Seems like we need to show inspection warning somewhere in `use` section.

P.S: Find Usages & resolve should also somehow support such use cases

Reply:
Inspection highlighting isn't very convenient - all lines of `use rule` are highlighted, other errors text offset not clear. Is better to make it narrow, e.g. highlight only the first line of use rule declaration, e.g. `use rule foo_ as foo1` or just `foo1` or etc:
![image](https://user-images.githubusercontent.com/72933/136110221-f4788ddc-6068-4b8c-a30e-7aa2077bafe7.png)

Reply:
ok, now works